README.md

```diff
@@ -146,6 +158,8 @@ branches:
         strict: true
         # Required. The list of status checks to require in order to merge into this branch
         contexts: []
+      # Commits pushed to matching branches must have verified signatures. Set to false to disable.
+      required_signatures: true
```

Comment:
i dont find this attribute mentioned in the docs at https://docs.github.com/en/rest/reference/repos#update-branch-protection

did you find details about this one somewhere else?

Reply:
found it on https://docs.github.com/en/rest/reference/repos#create-commit-signature-protection

and although i am writing it in this comment, **all new addition have been tested**.

Reply:
You can also find earlier confirmation of @foolioo [in this comment](https://github.com/probot/settings/issues/220#issuecomment-613741695)